README.md

```diff
@@ -41,6 +41,11 @@ docsearch({
   host: "YOUR_HOST_URL",
   apiKey: "YOUR_SEARCH_API_KEY",
   indexUid: "YOUR_INDEX_UID",
+  hotKeys: {
+    // set false to disable
+    ctrlWithKey: "k",
+    singleKeys: ["s", "/"],
+  }
```

Comment:
Let's remove this, I think we should keep the examples as concise and the defaults are good enough for most usecases.